cdk/database/__init__.py

```diff
@@ -118,3 +122,49 @@ def dynamodb_users_table(self):
                                                   type=aws_dynamodb.AttributeType.STRING),
                                               billing_mode=aws_dynamodb.BillingMode.PAY_PER_REQUEST,
                                               time_to_live_attribute="last_updated")
+
+    def dynamodb_quiz_progress_table(self):
+        """
+        This table will hold all users quiz progression.
+
+        Reason for using 2 tables for quiz data (quiz progression and quiz_list): 
+            - Easier to add new quizzes in the future
+            - Makes retrieving all quiz data for a user easier
+
+        schema:
+        - PK = clemson-username
+        - SK = quiz_id
+
+        """
+        self.quiz_progress_table = aws_dynamodb.Table(self,
+                                                      self.quiz_progress_id,
+                                                      point_in_time_recovery=True,
+                                                      removal_policy=core.RemovalPolicy.RETAIN,
+                                                      partition_key=aws_dynamodb.Attribute(
+                                                          name="username",
+                                                          type=aws_dynamodb.AttributeType.STRING
+                                                      ),
+                                                      sort_key=aws_dynamodb.Attribute(
+                                                          name="quiz_id",
+                                                          type=aws_dynamodb.AttributeType.STRING
+                                                      ),
+                                                      billing_mode=aws_dynamodb.BillingMode.PAY_PER_REQUEST,
+                                                      time_to_live_attribute="last_updated")
```

Comment:
Whats the purpose of the TTL on this table?

Reply:
I copy pasted from another table creation. TTL is an expiration date for the data in the table. I don't think that quiz_progress data should expire and quiz_list definitely doesn't need it.

Reply:
What does `time_to_live_attribute="last_updated"` mean?